v2/internal/testcommon/test_recorder.go

```diff
@@ -16,12 +16,7 @@ import (
 )
 
 // createTestRecorder returns an instance of testRecorder to allow recording and playback of HTTP requests.
-func createTestRecorder(
-	cassetteName string,
-	cfg config.Values,
-	recordReplay bool,
-	log logr.Logger,
-) (vcr.Interface, error) {
+func createTestRecorder(cassetteName string, cfg config.Values, recordReplay bool, log logr.Logger, hideCustomData map[string]string) (vcr.Interface, error) {
```

Comment:
Please keep long signatures with one parameter per row, it's a lot easier to read.